### ISSUE TITLE
Update in BULK_INCLUDE_TABLES_MAP needed for key "gsgk"

### ISSUE THREAD

**Issue by mybananas** [OpenEnergyPlatform/open-MaStR#455]
## Description of the issue

At least since August '23 is the mapping of the dictionary `BULK_INCLUDE_TABLES_MAP` in file `utils\constants.py`
is incorrect, i.e. needs an update.

## Ideas of solution

in file `utils\constants.py`
replace the lines 85/86
````
    "gsgk": [
        "anlageneeggeosolarthermiegrubenklaerschlammdruckentspannung",
        "einheitengeosolarthermiegrubenklaerschlammdruckentspannung",
    ],
````
with 
````
    "gsgk": [
        "anlageneeggeothermiegrubengasdruckentspannung",
        "einheitengeothermiegrubengasdruckentspannung",
    ],
````

## Context and Environment
* Version used:  0.13.1
* Operating system: linux/windows
* Environment setup and (python) version: python 3.10

## Workflow checklist
- [x] I am aware of the workflow in [CONTRIBUTING.md](https://github.com/OpenEnergyPlatform/open-MaStR/blob/production/CONTRIBUTING.md)

**Comment by FlorianK13** [comment]
Thanks @mybananas for opening the issue. I'll try to reproduce the error.

**Comment by FlorianK13** [comment]
@mybananas the issue is fixed and PR #456 will resolve it - thanks for your help!
